docs/transactions/gov/README.md

```diff
@@ -0,0 +1,91 @@
+# Gov
+
+This package creates transaction payloads with messages from the [Gov Module](https://docs.cosmos.network/v0.47/modules/gov).
```

Comment:
Nit
```suggestion
This package creates transaction payloads with messages from the [Governance Module](https://docs.cosmos.network/v0.47/modules/gov).
```

Reply:
Kept as is to maintain parity with the Cosmos SDK's naming (which is Gov).